adal/src/main/java/com/microsoft/aad/adal/AuthenticationRequest.java

```diff
@@ -63,6 +63,8 @@
 
     private boolean mIsExtendedLifetimeEnabled = false;
 
+    private String mTelemetryRequestId;
```

Comment:
no, correlation id can be set by the developer.

---
In reply to: [76160005](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76160005) [](ancestors = 76160005)